lib/ui/text.dart

```diff
@@ -1287,6 +1287,35 @@ enum BoxWidthStyle {
     max,
 }
 
+/// Where to vertically align the placeholder relative to the surrounding text.
+enum PlaceholderAlignment {
+  /// Match the baseline of the placeholder with the baseline.
+  baseline,
```

Comment:
this should mention the other argument to addPlaceholder needing to be non-null in this case

Reply:
the next two too